sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py

```diff
@@ -14,7 +14,7 @@ class EUTimelineSpider(scrapy.Spider):
     name = 'eu-timeline'
     base_url = 'https://ec.europa.eu'
     url = 'https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/timeline-eu-action_en'
-    presscorner_base_url = 'https://ec.europa.eu/commission/presscorner/detail'
+    presscorner_base_url = 'https://ec.europa.eu'
```

Comment:
I still do not understand why we need this. Can we delete this line? 
If not, then why?

Reply:
which line? base_url?